internal/service/autoscaling/policy.go

```diff
@@ -87,6 +87,276 @@ func ResourcePolicy() *schema.Resource {
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									"customized_scaling_metric_specification": {
+										Type:          schema.TypeList,
+										Optional:      true,
+										MaxItems:      1,
+										ConflictsWith: []string{"predictive_scaling_configuration.0.metric_specification.0.predefined_scaling_metric_specification"},
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"metric_data_queries": {
+													Type:     schema.TypeList,
+													Required: true,
+													MaxItems: 10,
```

Comment:
is there a link to the API reference for `MaxItems` for `metric_data_queries`? It doesn't seem to be specified here: https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PredictiveScalingCustomizedCapacityMetric.html

Reply:
Good question. I think we are missing public documentation for it. I will contact our tech writer to fill in this gap.

Reply:
It is actually mentioned somewhere else in user guide https://docs.aws.amazon.com/autoscaling/ec2/userguide/predictive-scaling-customized-metric-specification.html#custom-metrics-limitations